### ISSUE TITLE
Include Design Component version in Design Toolbox App

### ISSUE THREAD

**Issue by B3nz01d** [Orange-OpenSource/ouds-ios#760]
## User Story
**As a** User of the DEsign Toolbox app
**I want** to know what version of the component has been implemented
**So that** I can check if the current implemented component in the dev library is corresponding to the current figma version in the design library.

## Description
Within the Demo app, for each component that has been implemented until today (and will be in the future), a new section needs to be added at the end of the component demo page to show:
"Ref Design Verison: v1.1.2"

**Comment by pylapp** [comment]
Partially implemented in commit 0f6aaf83568662896cbacd8dcd06e89a5c47499f, wording should be improved

**Comment by B3nz01d** [comment]
see in #848
